compiler-rt/lib/radsan/radsan.h

```diff
@@ -0,0 +1,76 @@
+//===--- radsan.h - Realtime Sanitizer --------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "sanitizer_common/sanitizer_internal_defs.h"
+
+extern "C" {
+
+/**
+    Initialise radsan interceptors. A call to this method is added to the
+    preinit array on Linux systems.
+
+    @warning Do not call this method as a user.
+*/
+SANITIZER_INTERFACE_ATTRIBUTE void radsan_init();
+
+/** Enter real-time context.
```

Comment:
Let's replace `real-time` in this header with `nonblocking`?

Reply:
not sure either. There is the attribute that is nonblocking, but we are the realtime sanitizer. I do still think this is a real time context that we are describing. I am fine either way, both work IMO

Reply:
Good point. Let's leave it as is for now and change it if the reviewers prefer `radsan_nonblocking_enter`